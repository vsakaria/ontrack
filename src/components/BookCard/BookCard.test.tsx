import React from "react";
import { cleanup, render } from "@testing-library/react";
import { BookCard } from ".";

afterAll(cleanup);

test("BookCard should render the correct DOM", () => {
  const book = {
    "book_title": "SOME_STRING",
    "book_author": "SOME_STRING",
    "book_publication_year": "SOME_STRING",
    "book_pages": "SOME_STRING"
  }

  const { container } = render(<BookCard book={book} />);
  expect(container).toMatchSnapshot("SearchField");
})