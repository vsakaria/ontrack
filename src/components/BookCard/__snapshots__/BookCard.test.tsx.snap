// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookCard should render the correct DOM: SearchField 1`] = `
<div>
  <div
    class="mb-4 mr-4 card"
    style="width: 30rem;"
  >
    <div
      class="card-body"
    >
      <div
        class="card-title h5"
      >
        SOME_STRING
      </div>
      <div
        class="mb-2 card-subtitle h6"
      >
        by: SOME_STRING
      </div>
      <p
        class="card-text"
      >
        date: 
        SOME_STRING
      </p>
      <p
        class="card-text"
      >
        pages: 
        SOME_STRING
      </p>
    </div>
  </div>
</div>
`;
